src/nvhttp.cpp

```diff
@@ -1033,7 +1033,13 @@ namespace nvhttp {
 
     // Verify certificates after establishing connection
     https_server.verify = [add_cert](SSL *ssl) {
-      crypto::x509_t x509 { SSL_get_peer_certificate(ssl) };
+      crypto::x509_t x509 {
+#if OPENSSL_VERSION_MAJOR >= 3
```

Comment:
I don't believe any builds still using anything less than 3.0, are you wanting to just keep this check for any who may self compile with an older version?

Reply:
Yeah, it's such a trivial change that it doesn't seem like a problem to keep supporting 1.1 and 3.0+.